teachers_digital_platform/js/scroll.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const scroll = require( './scroll.js' );
+
+const HTML_SNIPPET = `
+  <a class="one" href="#one" data-scroll>one</a>
+  <a class="two" href="#two" data-scroll>two</a>
+  <a class="three" href="#three" data-scroll>three</a>
+  <div id="one"></div>
+  <div id="two"></div>
+  <div id="three"></div>
+`;
+
+// Helper function to simulate events
+function simulateEvent( eventType, target, eventOption ) {
+  const event = document.createEvent( 'Event' );
+  if ( eventOption && eventOption.keyCode ) {
+    event.keyCode = eventOption.keyCode;
+  }
+  event.initEvent( eventType, true, true );
+  return target.dispatchEvent( event );
+}
```

Comment:
This helper function is copied from [cfgov-refresh](https://github.com/cfpb/cfgov-refresh/blob/8c780180d5352fcd4d7f7ee63aee343fc8f91e9f/test/util/simulate-event.js). If you end up using it a lot you may want to move it into a reusable file.